src/alpine/.devcontainer/devcontainer.json

```diff
@@ -1,16 +1,19 @@
 {
 	"name": "Alpine",
-	"image": "mcr.microsoft.com/devcontainers/base:alpine-${templateOption:imageVariant}",
+	"image": "mcr.microsoft.com/devcontainers/base:alpine-${templateOption:imageVariant}"
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
```

Comment:
Would it make sense to instead or additionally point to https://containers.dev/implementors/features/?